backend/schema/metadata.ts

```diff
@@ -0,0 +1,31 @@
+export const BRANDS = {
```

Comment:
Can brand just be a string?

Reply:
I think we’ll want it to be more of an enum. That way we can write brand-specific logic (see the canonical model number bit in pipeline/ for an example) without having to worry about string typos causing bugs. 

RA likes to do this; many of their fields like this are much larger enums (many hundreds of values).